Gemfile

```diff
@@ -31,5 +31,6 @@ end
 
 group :test do
   gem 'capybara', '>= 2.15', '< 4.0'
+  gem 'minitest-stub_any_instance'
```

Comment:
We recently merged https://github.com/cedarcode/webauthn-rails-demo-app/pull/87 to `master`.

Should we use `"~> 1.0"` here?